### PR TITLE
Add meet and greet charts

### PR DESCRIPTION
**Pull request by ilayda21**
## Done
Meet and greet interview stage's charts are added

## QA
- Run: `dotrun`
- Open `http://0.0.0.0:8501/stages/meet-and-greet-interview`
- Verify charts are displayed correctly.

## Issue
Fixes #186

### CODE REVIEW

File: webapp/stages/services/interview.py [hunk 1]
```diff
@@ -18,6 +18,15 @@ def _get_interviews_scheduled(app_id):
     return api.get_interviews_scheduled(app_id)
 
 
+def _find_interviews_in_stage(interviews, type):
```
Comment (andesol) [inline webapp/stages/services/interview.py:21]: I deleted this function, so apparently shouldn't be there

File: static/js/shared/utils/retrievePendingInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+export const retrievePendingInfo = async (endpoint: string, stageName: string, queryID: string, queryInfo: QueryResult) => {
+    const generateQuery = (data: any[]) => {
+        let query = "";
+        data.forEach((jobInfo: any) => {
+          const applications = jobInfo.applications
+            .filter(
+              (application: any) =>
+                application["current_stage"] === stageName &&
+                application["has_take_home_test"]
```
Comment (andesol) [inline static/js/shared/utils/retrievePendingInfo.tsx]: I essence, there's no need to check for the take home test in the interview stage. But I guess you're doing it in order to keep the function general, so it's ok. It shouldn't affect the results.

**Review by andesol — APPROVED** [review]

File: static/js/shared/utils/retrievePendingInfo.tsx (hunk 2 above)
Comment (ilayda21) [reply static/js/shared/utils/retrievePendingInfo.tsx]: yes if it does not matter for interview stages i just want to leave it

File: webapp/stages/services/interview.py (hunk 1 above)
Comment (ilayda21) [reply webapp/stages/services/interview.py]: done :+1:

**Pull request merged**